api/securitytest/spotbugs.go

```diff
@@ -170,6 +172,7 @@ func (spotbugsScan *SecTestScanInfo) prepareSpotBugsVulns() {
 			spotbugsVuln.Code = fmt.Sprintf("Code beetween Line %s and Line %s.", startLine, endLine)
 			spotbugsVuln.Line = startLine
 			spotbugsVuln.File = spotbugsOutput.SpotBugsIssue[i].SourceLine[j].SourcePath
+			spotbugsVuln.Title = spotbugsVuln.Details + " in " + spotbugsVuln.File
```

Comment:
Example output:

```
[HUSKYCI][!] Title: PREDICTABLE_RANDOM in com/mycompany/app/App.java
[HUSKYCI][!] Language: Java
[HUSKYCI][!] Tool: SpotBugs
[HUSKYCI][!] Severity: MEDIUM
[HUSKYCI][!] Confidence: MEDIUM
[HUSKYCI][!] Details: PREDICTABLE_RANDOM
[HUSKYCI][!] File: com/mycompany/app/App.java
[HUSKYCI][!] Line: 12
[HUSKYCI][!] Code: Code beetween Line 12 and Line 12.
[HUSKYCI][!] Type: SECPR
```

Unfortunately, the SpotBugs output is not that verbose and I think we can use only its details to avoid using filenames in the title. What is your opinion about this one? 🙃

```
spotbugsVuln.Title = spotbugsVuln.Details
```